### DISCUSSION TITLE
Search Line limits were exceeded, some search paths have been omitted RKE2

### DISCUSSION THREAD

**Discussion by samavedulark** [rancher/rke2#3216] [General]
I have installed RKE2 version (v1.21.14+rke2r1), kubernetes is working fine, i am able to see nodes.
But when i installed one helm chart, one of the pod is pending state.

```
Warning  DNSConfigForming  10s (x72 over 45m)  kubelet            Search Line limits were exceeded, some search paths have been omitted, the applied search line is: gpu-operator.svc.cluster.local svc.cluster.local cluster.local itx rbd itm
```

I checked my resolv.conf and i see in search line 3 entires.

My question is is editing resolv.conf is the only way, or is there any alternative steps needs to be done.

**Comment by brandond** [comment]
That is not why your pod is stuck pending. That's just a warning that you have too many `search` entries in you resolv.conf, and the kubelet had to remove some of them. While editing your resolv.conf would make that warning go away, that warning is not something that would cause your pod not to run. I would probably look at the `describe pod` output and kubelet/containerd logs in more detail to figure out why it's not running.
